{{Waterloo}}
<html>
<div id ="mainContainer">
<div id ="contentContainer">

    <h1>CRISPieR: re-engineering CRISPR-Cas9 with functional applications in eukaryotic systems</h1>

    <section id="motivation" title="Motivation">
        <h2>Motivation</h2>
        <p>
            The CRISPR-Cas9 system is an astounding tool with many applications in synthetic biology. Since its popularization a few years ago, CRISPR papers have flooded major journals and many iGEM teams have worked to improve its use as a genome editing tool. The 2015 Waterloo team is taking a three-pronged approach to expand upon previous research for wider, more efficient, and more flexible use of CRISPR-Cas9.
        </p>
        <p>
            We aim to make target selection and guide sequence replacement physically easier in the lab. To target a DNA sequence, a single guide RNA (sgRNA) is used to identify a match. Each sgRNA can only have one intended target, however, which forces us to re-synthesize and re-clone the entire sgRNA sequence for each new target we'd like to test. By being able to swap out a 20 base pair section of the sgRNA sequences, instead of synthesizing new targets from scratch, we hope to reduce the turnaround time for using CRISPR to target different sequences.
        </p>
        <p>
            By building on recent papers, such as Kleinstiver et al., 2015, we are trying to make Cas9’s binding to a protospacer adjacent motif (PAM) site more flexible. The standard S. pyogenes type II CRISPR-Cas9 binds to a PAM of length 3, namely NGG. While this is fairly general, requiring that this sequence be next to the target sequence limits where Cas9 can cut. We will produce a model that suggests Cas9 variants and their preferred PAM sites, and we will attempt to demonstrate this model’s validity. The overarching goal is to create Cas9 variants that will bind to any desired PAM site, and we hope to take some steps forward to make that goal more achievable and directed.
        </p>
        <p>
            Finally, CRISPR originated as a viral defence mechanism for bacteria, for specific and targeted immunity. Multi-cellular organisms have developed their own defenses to achieve this goal, but from our research it appears that groups have not publicly attempted to introduce CRISPR as an antiviral mechanism in multicellular organisms. Our team is attempting to use the CRISPR system in plants to discover whether it can defend against a class of double-stranded DNA viruses.
        </p>
    </section>

    <section id="sgRNA" title="sgRNA Modification">
        <h2>sgRNA Modification</h2>
        <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
    </section>

    <section id="cas9" title="Cas9 Modification">
        <h2>Cas9 Modification</h2>
        <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
    </section>

    <section id="plants" title="Plant Defence">
        <h2>Plant Viral Defence</h2>
        <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
    </section>


    <div class="prevHighlightBox">
        <h5>What should this page contain?</h5>
        <ul>
            <li> A clear and concise description of your project.</li>
            <li>A detailed explanation of why your team chose to work on this particular project.</li>
            <li>References and sources to document your research.</li>
            <li>Use illustrations and other visual resources to explain your project.</li>
        </ul>


        <br />
        <h4>Advice on writing your Project Description</h4>

        <p>We encourage you to put up a lot of information and content on your wiki, but we also encourage you to include summaries as much as possible. If you think of the sections in your project description as the sections in a publication, you should try to be consist, accurate and unambiguous in your achievements.</p>

         <p>Judges like to read your wiki and know exactly what you have achieved. This is how you should think about these sections; from the point of view of the judge evaluating you at the end of the year.</p>


        <br />
        <h4>References</h4>
        <p>iGEM teams are encouraged to record references you use during the course of your research. They should be posted somewhere on your wiki so that judges and other visitors can see how you though about your project and what works inspired you.</p>



        <h4>Inspiration</h4>
        <p>See how other teams have described and presented their projects: </p>

        <ul>
            <li><a href="http://2014.igem.org/Team:Imperial/Project"> Imperial</a></li>
            <li><a href="http://2014.igem.org/Team:UC_Davis/Project_Overview"> UC Davis</a></li>
            <li><a href="http://2014.igem.org/Team:SYSU-Software/Overview">SYSU Software</a></li>
        </ul>
    </div>

</div>
</div>
</html>
{{Waterloo_Footer}}
